packages/mds-stream/test-utils.ts

```diff
@@ -0,0 +1,14 @@
+import Sinon from 'sinon'
+import { WriteStream } from './types'
+
+export const mockStream = <T>(stream: WriteStream<T>) => {
```

Comment:
Would be cool if you could pass in an `options` or `overrides` object where you could supply methods for `initialize, shutdown, write`. E.g. `{ initialize: () => throw new Error(...) }` if you wanted to test for initialize failures, for example